tests/gis_tests/tests.py

```diff
@@ -94,3 +94,19 @@ def test_no_version_number(self):
         ops = FakePostGISOperations()
         with self.assertRaises(ImproperlyConfigured):
             ops.spatial_version
+
+    def test_version_dependent_funcs(self):
+        """
+        Test that PostGIS version dependent functions are switched accordingly.
```

Comment:
I've added the "Remove when dropping support for PostGIS 2.1", but feel that it should be unnecessary. The logical place for this text would be as a comment in `PostGISOperations.function_names()`, but that code is so obviously useless after dropping PostGIS 2.1 support and easily locatable when searching for `spatial_version` which everybody should do when raising the minimum `spatial_version` supported. Furthermore, once `function_names` has been changed back to a regular property to drop PostGIS <2.2 support, `test_version_dependent_funcs` will fail and it will be obvious that it is only needed to test backwards-compatibility with PostGIS <2.2. — But, you know better which hints for future maintenance you appreciate :-)